SpanishFilter/sections/adservers.txt

```diff
@@ -4,6 +4,8 @@
 ! https://forum.adguard.com/index.php?threads/ivoox-com-podcasts-issues.29530/
 ! ||addevweb.com^$third-party
 !
+||choconart.com^$third-party
```

Comment:
It's `clickadu` domain, so I think it could be added to Base/English filter.
By the way, on which website did you find `engagedhits.com`?

Reply:
`https://lk.clickhouse.xyz/hPfbJY` Here. But the link expired.